src/util/rewriteUrls.ts

```diff
@@ -1,6 +1,6 @@
 import * as urlParser from 'url';
 import { isMirrored } from './saveArticles';
-import { migrateChildren, getMediaBase, getFullUrl, getRelativeFilePath } from './misc';
+import { migrateChildren, getMediaBase, getFullUrl, getRelativeFilePath, encodeArticleId} from './misc';
```

Comment:
A space is missing here after `encodeArticleId`.

Reply:
Done